cmd/policy-main.go

```diff
@@ -444,13 +444,13 @@ func mainPolicy(ctx *cli.Context) error {
 	switch ctx.Args().First() {
 	case "list":
 		// policy list alias/bucket/prefix
-		runPolicyListCmd(ctx)
+		runPolicyListCmd(ctx.Args().Tail())
 	case "links":
 		// policy links alias/bucket/prefix
-		runPolicyLinksCmd(ctx)
+		runPolicyLinksCmd(ctx.Args().Tail(), ctx.Bool("recursive"))
 	default:
 		// policy [download|upload|public|] alias/bucket/prefix
-		runPolicyCmd(ctx)
+		runPolicyCmd(ctx.Args().Tail())
```

Comment:
You should not Tail() this, the first argument is the access permission (download, upload, public)

Reply:
Fixed